deploy/charts/library/templates/_recommended-labels.tpl

```diff
@@ -1,7 +1,13 @@
 {{- /*
-  Common labels
+  Modern recommended labels following Kubernetes best practices
+  Based on: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
 */}}
 {{- define "library.rook-ceph.labels" -}}
+operator: rook
+storage-backend: ceph
+app.kubernetes.io/name: rook-ceph
+app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
How about suppressing this as well in common.yaml? This doesn't make much sense:
```
app.kubernetes.io/instance: release-name
```